### PR TITLE
Story validation

### PR DESCRIPTION
**Pull request by luceleaftea**
Test steps:
1. For both add and edit pages...
2. Try saving with various combinations of forms unfilled
3. It should refuse to submit and highlight necessary forms if the title and/or body is missing

Note: Check the navigation if you refresh on a failed validation page - does it navigate to the correct page?

### CODE REVIEW

File: app/assets/stylesheets/stories.scss [hunk 1]
```diff
@@ -0,0 +1,7 @@
+// A little hacky, sorry!  Change this if you ever change the order of the form fields
+// It should target the WYSIWYG editor to make it full width on failed validations
+.field:nth-of-type(3) {
```
Comment (chiperific) [inline app/assets/stylesheets/stories.scss]: I believe in you. You can do this.

File: config/environments/production.rb [hunk 2]
```diff
@@ -18,7 +18,7 @@
 
   # Ensures that a master key has been made available in either ENV["RAILS_MASTER_KEY"]
   # or in config/master.key. This key is used to decrypt credentials (and other encrypted files).
-  # config.require_master_key = true
+  config.require_master_key = true
```
Comment (chiperific) [inline config/environments/production.rb]: This is the wrong way. Production needs to rely on ENV vars.

File: config/environments/development.rb [hunk 3]
```diff
@@ -63,6 +63,7 @@
   # number of complex assets.
   config.assets.debug = true
 
+  config.require_master_key = true
```
Comment (chiperific) [inline config/environments/development.rb]: Zach says this can be commented out. And he's right.

File: db/schema.rb [hunk 4]
```diff
@@ -37,8 +37,6 @@
     t.integer "people_goal"
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
-    t.index ["end_date", "start_date"], name: "between_end_start_dates"
```
Comment (chiperific) [inline db/schema.rb]: These should exist. I'm not sure why your branch doesn't have them. Maybe un-track this file?

Comment (chiperific) [reply db/schema.rb]: Same for all in file.

File: config/environments/production.rb (hunk 2 above)
Comment (luceleaftea) [reply config/environments/production.rb]: This must have accidentally been changed when I rebased - will flip it back!

**Pull request merged**